src/main/java/com/algorand/algosdk/transaction/Transaction.java

```diff
@@ -729,10 +747,12 @@ public Transaction(
                         // keyreg fields
                         ParticipationPublicKey votePK,
                         VRFPublicKey vrfPK,
+                        MerkleVerifier stateProofKey,
                         BigInteger voteFirst,
                         BigInteger voteLast,
                         // voteKeyDilution
                         BigInteger voteKeyDilution,
+                        boolean nonpart,
```

Comment:
Adding new args to this constructor breaks backwards compatibility. Unfortunately we didn't notice that when we added a new option for extra pages, but we shouldn't make that mistake again.

You could make a new constructor with the new args, or add them to this constructor at the end and give them a default value to maintain backwards compatibility. But probably the simplest solution is to not add them at all, since the main purpose of this constructor seems to be to make the deprecated txn constructor methods here work, and if you don't change `createKeyRegistrationTransaction`, there's no need to change this.